packages/suite-native/src/actions/suite/routerActions.ts

```diff
@@ -88,8 +92,17 @@ export const init = () => (dispatch: Dispatch, getState: GetState) => {
  * @param {boolean} preserveParams
  */
 export const goto =
-    (routeName: Route['name'], params?: RouteParams, _preserveParams = false) =>
+    (
+        routeName: Route['name'],
+        options: {
+            params?: RouteParams;
+            preserveParams?: boolean;
+            anchor?: AnchorType;
+        } = {},
+    ) =>
     (dispatch: Dispatch) => {
+        const { params, preserveParams: _preserveParams, anchor: _anchor } = options;
```

Comment:
I don't see the point of extracting those options here if they are not used.
```suggestion
        const { params } = options;
```